### DISCUSSION TITLE
Shortcut to Jump to Command Line?

### DISCUSSION THREAD

**Discussion by Smoghog** [winder/Universal-G-Code-Sender#1568] [Help] [answered]
I was wondering if there is a shortcut/hotkey to put the cursor immediately into the command line? I am hoping to create some macros on a keyboard that have to be entered in the command line, and would like to be able to tell UGS to put the cursor there first, then proceed with the macro. If not, is this something that could be added to the custom hotkey creation screen?

**Comment by breiler** [comment, marked as answer]
Yes!

Open the shortcut settings and assign a shortcut to Console Window, that will give focus to the command text field:
![image](https://user-images.githubusercontent.com/8962024/112176746-a74e2a00-8bf8-11eb-8a02-54376254d507.png)

(I needed to restart UGS for it to take effect)

**Reply by Smoghog** [reply]
Interesting, I tried that and it didn't work for me. It does change the focus to the console window, but it doesn't automatically put the cursor in the entry field. I'm using v 2.0.7, December 20 snapshot

**Reply by breiler** [reply]
Ah, it was added Jan 3: #1502
So it is available in the nightly build

**Reply by Smoghog** [reply]
Ah, perfect! I'll get the latest nightly now and keep tinkering. Thank you!!!

**Comment by Serkankarakus-blip** [comment]
during the first installation phase of the application, I get this error "Cannot be run from folder that contains non-ASCII characters in path", can you help me do what I need to do?
